OHS-Hosting-Infrastructure/complaint-tracker
# changes shape of fake data to match current HSES API expectation

## Description of change

Aligns our fake data structure with the latest spec from HSES

notable changes:
- HSES uses "summary" instead of "issue", changed in our code to reflect that
- the FORMATTED_STATUS in `model/complaint.rb` diverges from the info
  we are being sent by HSES for status
- region now an integer, not a string

## Acceptance Criteria

Matches the `issues_openapi.yml` structure and data types

## How to test

You may want to actually open up the yaml file from Uira and check that I've got all the right potential values and structure in this code.

## Issue(s)

* closes https://github.com/OHS-Hosting-Infrastructure/complaint-tracker/issues/103


## Checklist

To be completed by the submitter:


- ~Meets accessibility standards (WCAG 2.1 Levels A, AA)~
- ~API Documentation updated~
- ~Boundary diagram updated~
- ~Logical Data Model updated~
- ~[Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

## To the Reviewer

This project is using [Conventional Comments](https://conventionalcomments.org/) to give structure
and context to PR comments. Please use these labels in your comments.

* **praise:**
* **nitpick:**
* **suggestion:**
* **issue:**
* **question:**
* **thought:**
* **chore:**
